.github/workflows/aws.yml

```
@@ -16,10 +20,10 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_ACCESS_KEY }}
-          aws-region: us-east-1
+          aws-region: ${{ secrets.REGION }}
       - name: Package and Deploy to AWS CloudFormation
         run: |
-          aws cloudformation package --template-file lambda-loader/template.yaml \
+          aws cloudformation package --template-file ./lambda-loader/template.yaml \
```

```suggestion
          mkdir dist
          cp -r src/ dist/
          aws cloudformation package --template-file ./lambda-loader/template.yaml \
```
@JimLiu0 If it helps